src/sentry/migrations/0388_create_groupforecast_table.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.2.28 on 2023-03-20 16:35
+
+import django.db.models.deletion
+import django.utils.timezone
+from django.db import migrations, models
+
+import sentry.db.models.fields.array
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0387_add_until_escalating_column_to_groupsnooze"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="GroupForecast",
```

Comment:
Once this is merged could this model also be added to the relations [for group deletion](https://github.com/getsentry/sentry/blob/master/src/sentry/deletions/defaults/group.py#L12)